docs/BrowserStackLocal_README.md

```diff
@@ -0,0 +1,46 @@
+# Setting Up Browser-Stack with Teswiz:
+
```

Comment:
Add a small paragraph why this is necessary, or what is the use case to refer to this post.